quickjs-libc.c

```diff
@@ -140,14 +135,22 @@ typedef struct {
     size_t sab_tab_len;
 } JSWorkerMessage;
 
+typedef struct JSWaker {
+#ifdef _WIN32
+    HANDLE handle;
+#else
+    int read_fd;
+    int write_fd;
+#endif
+} JSWaker;
+
 typedef struct {
     int ref_count;
 #ifdef USE_WORKER
```

Comment:
I know you didn't introduce this, but how about ifdef-ing the whole JSWorkerMessagePipe out?

Reply:
I'll try that but I fully expect it to break the wasi build _again_.